src/server/parser.ts

```diff
@@ -35,13 +35,13 @@ function startIndex() {
   )
 
   scanProcess.on('message', (mess) => {
-    const { data, log } = mess
+    const { data, msglog } = mess
     if (data) {
       if (!workspace.isExistsBuffer(data.uri)) {
         workspace.updateBuffer(data.uri, data.node)
       }
     }
-    if (log) {
+    if (mess.log) {
```

Comment:
As a beginner I have no idea why using `msglog` did not wok there.

Reply:
should also change the name which scan emit